packages/actor-http-native/lib/Requester.ts

```diff
@@ -68,24 +68,25 @@ export default class Requester {
         response.pipe(decoded);
         // Copy response properties
         decoded.statusCode = response.statusCode;
-        // Wrap headers into an header object type
-        const responseHeaders: any = new Headers();
-        for (const header in response.headers) {
-          responseHeaders[header[0]] = header[1];
-        }
-        decoded.headers = responseHeaders;
+        decoded.headers = this.convertFetchHeadersToHash(response);
```

Comment:
I would just pass `response.headers`. `convertFetchHeadersToHash` does not need to be aware of the full `response` object. Also simplifies testing.

Reply:
Zou je dan ook willen dat ik daar apart een test voor toevoeg?

Reply:
Sure, but nothing to extensive though. Just an empty and non-empty conversion check should be fine.

Reply:
A test has been added. But as seen as in the new code there are now 2 elements passed from response. So now the entire response is again added. Unsure if thats still unclean. But there are passing issues if response.input.headers are passed into the function that's another reason as to why i'm passing the whole object now.

Reply:
Hrmm, this feels wrong. It's still inclear to me why `response.input.headers` is needed.